manager/testdata/new-blueprints/scenario6/plotter.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: Plotter
+metadata:
+  name: app1-ns1 # plotter name is a combination of app and namespace
+  namespace: m4d-system
+  labels:
+spec:
+  appSelector: # Selector of the application that uses this workload
+    clusterName: thegreendragon
+    workloadSelector:
+      matchLabels:
+        app: demoapp
+  assets:
+  - assetId: "m4d-notebook-sample/paysim"  # dataset requested by user
+    credentials:
+      read:
+        vault:
+          address: http://vault.m4d-system:8200
+          authPath: /v1/auth/kubernetes/login
+          role: module
+          secretPath: /v1/kubernetes-secrets/paysim?namespace=m4d-notebook-sample
+    connection:
+      s3:
+        endpoint: localhost:8001
+        bucket: srcbucket
+        object: data.csv
+    dataformat: csv
+  - assetId: "m4d-notebook-sample/paysim-temp"  # allocated by fybrik control plane
+    advertisedAssetId: "m4d-notebook-sample/paysim" # Id that is used by user facing services
+    credentials:
+      read:
+        vault:
+          address: http://vault.m4d-system:8200
+          authPath: /v1/auth/kubernetes/login
+          role: module
+          secretPath: /v1/kubernetes-secrets/paysim-temp?namespace=m4d-notebook-sample
+      write:
+        vault:
+          address: http://vault.m4d-system:8200
+          authPath: /v1/auth/kubernetes/login
+          role: module
+          secretPath: /v1/kubernetes-secrets/paysim-temp?namespace=m4d-notebook-sample
+    connection:
+      s3:
+        endpoint: localhost:8001
+        bucket: destbucket
+        object: data.parquet
+    dataformat: parquet
+  flows:
+  - name: flow-paysim
+    flowType: read
+    assetId: "m4d-notebook-sample/paysim"
+    subflows:
+    - name: subflow-paysim-copy
```

Comment:
Should this be -- to indicate that the copy and the arrow flight are serial?  What do you think @froesef ?

Reply:
Jointly decided no